src/mlagility/cli/report.py

```diff
@@ -1,521 +1,204 @@
 import os
 import csv
-import dataclasses
-from typing import List
-import yaml
+import json
+from datetime import datetime
+from pathlib import Path
+from dataclasses import dataclass
 import groqflow.common.printing as printing
 import groqflow.common.build as build
 import groqflow.common.cache as cache
-from groqflow import groqmodel
+from groqflow.groqmodel import groqmodel
+from mlagility.common import labels
 
 
-def _successCleanup(item):
-    return item if item is not None else False
+def _update_numeric_attribute(new_val, current_val, default="-"):
+    """
+    Updates a numeric attribute if needed
+    """
+    if current_val == default:
+        return new_val if new_val is not None else default
+    else:
+        return current_val
 
 
-def _numericCleanup(item):
-    return item if item is not None else "-"
+def get_estimated_groq_latency(model_folder, cache_folder):
+    """
+    Returns estimated Groq latency (io + compute - not including runtime) in ms
+    """
+    try:
+        gmodel = groqmodel.load(model_folder, cache_folder)
+        if (
+            gmodel.state.info.assembler_success  # pylint: disable=singleton-comparison
+            == True
+        ):
+            return 1000 * gmodel.estimate_performance().latency
```

Comment:
gmodel.estimate_performance().latency_unit is already `milliseconds`, right? So why multiply by 1000?

Reply:
Not really. It is in seconds. Here is one example:

`GroqEstimatedPerformance(pcie_input_latency=1.0032000000000002e-05, compute_latency=0.0009432577777777778, pcie_output_latency=1.8256000000000003e-05, latency_units='seconds', throughput_units='inferences per second')`

I'm raising an exception to guarantee that we get notified if anything changes